### PR TITLE
Add blueprint tests

### PR DESCRIPTION
**Pull request by jeradg**
Addresses https://github.com/san650/ember-cli-page-object/issues/124, https://github.com/san650/ember-cli-page-object/issues/100 and https://github.com/san650/ember-cli-page-object/issues/61.

### CODE REVIEW

**Comment by jeradg** [comment]
@san650 @juanazam: The blueprint tests are failing because `ember-cli-blueprint-test-helpers` requires us to use a newer version of `ember-cli`. Do you see an issue with upgrading the version of `ember-cli` in `package.json` to `^2.4.0`? We'll still be checking compatibility using `ember-try`, of course.

**Comment by juanazam** [comment]
@jeradg I don't see any problems

**Comment by san650** [comment]
@jeradg let's update ember-cli on another PR first and then rebase this PR on top of master. I can do this PR if you want.

**Comment by jeradg** [comment]
@san650: I have a branch where I'm working on the CLI update. Planning to merge that tomorrow if all goes well.

**Comment by jeradg** [comment]
@san650 and @juanazam: This PR is ready for review.

PR #155 fixed the main problem (blueprints not working in addons), so this PR is now focused on adding test coverage for blueprints.

Note that this PR also moves the addon tree test to the node-tests folder.

**Comment by san650** [comment]
@jeradg can you rebase with master and squash the commits?

**Comment by jeradg** [comment]
@san650: Done.

File: app/blueprint-tests/page-object-component.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-page-object/blueprint-tests/page-object-component';
```
Comment (san650) [inline app/blueprint-tests/page-object-component.js]: I think this file is not needed

File: app/blueprint-tests/page-object.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-page-object/blueprint-tests/page-object';
```
Comment (san650) [inline app/blueprint-tests/page-object.js]: This file is not needed also

Comment (jeradg) [inline app/blueprint-tests/page-object.js]: Hmm. I think those were added by ember-cli-blueprint-test-helpers. I'll check it out.

**Comment by jeradg** [comment]
@san650: Removed those unnecessary files.

**Comment by san650** [comment]
👏

**Comment by Turbo87** [comment]
@jeradg any reason for using the old blueprint testing syntax and semantics compared to the [new](https://github.com/ember-cli/ember-cli-blueprint-test-helpers#example-usage) one? it has been introduced with v0.11.0 and should work fine here.

**Comment by jeradg** [comment]
@turbo87: Only that I started the PR before v0.11.0 was released. A new PR using the v0.11 syntax would be welcome!

**Comment by Turbo87** [comment]
I see, thanks for the feedback!

**Pull request merged**